packages/react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -477,6 +477,10 @@ class ComboBoxInternal extends React.Component<IComboBoxInternalProps, IComboBox
    * {@inheritdoc}
    */
   public focus = (shouldOpenOnFocus?: boolean, useFocusAsync?: boolean): void => {
+    if (this.props.disabled === true) {
```

Comment:
I don't know if I caught this in the original PR but couldn't this be just

```suggestion
    if (this.props.disabled) {
```

?

Reply:
Yes you're right, it is basically the same, committing your suggestion. Thank you :)